flagext/register.go

```diff
@@ -53,3 +54,41 @@ func DefaultValues(rs ...interface{}) {
 	}
 	_ = fs.Parse([]string{})
 }
+
+// RegisteredFlagsTracker is an interface that allows to extract RegisteredFlags.
+type RegisteredFlagsTracker interface {
+	GetRegisteredFlags() RegisteredFlags
```

Comment:
Idiomatic Go is to drop the `Get` prefix on the getters.
```suggestion
	RegisteredFlags() RegisteredFlags
```

Reply:
Thank you, I have renamed the method.